frontend/model/contracts/shared/nativeNotification.js

```diff
@@ -18,11 +18,8 @@ export async function requestNotificationPermission (force: boolean = false): Pr
     }
   }
 
-  // TODO FIXME: TEMPORARILY DISABLED AS IT CAUSES AN ERROR DURING THE TESTS
-  if (isNaN(1) && Notification.permission === 'granted') {
+  if (Notification.permission === 'granted') {
     await sbp('service-worker/setup-push-subscription').catch((e) => {
-      // TODO: Temporary until this is better addressed. When running tests,
-      // it results in an AbortError.
```

Comment:
Actually, just removing that `isNaN(1)` made it start working again, and I didn't encounter any `AbortError` kind of error message while seeing it was running okay. My guess is that probably the similar issue was sorted out a while ago but then that fix didn't get merged to `e2e-protocol` branch and went to `master` instead.